lib/collection/src/operations/types.rs

```diff
@@ -1304,6 +1305,9 @@ pub struct VectorParams {
 
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub datatype: Option<Datatype>,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub multi_vec_config: Option<MultiVectorConfig>,
```

Comment:
We need some flag to say that these vectors are multivectors. But we found just `bool` a bad idea because multivectors may be configurable in future (for instance, we may introduce new multivector scorers, not only `MaxSim`)

Reply:
we need this field to enable multivector support when creating a collection